### DISCUSSION TITLE
fixed marker list and preferences window

### DISCUSSION THREAD

**Discussion by simonTx1** [g4ixt/QtTinySA#39] [Ideas]
Hi, 
Congratulation for the great work on the "fixed marker list" ! 
I've tested the "Import" and "Add Row" features in V0.10.1 and it works great.

Here is a sugestion or two to make it more user-friendly : 
For now, the import feature only allows appending markers to the table displayed above, and changing the "RF mic" list for a different one means wether having to delete rows one by one, or delete the personnalized configuration database 'QtTSAprefs.db' and replace it with the original. 
Would it be possible to : 
-  allow multiple row selection to be deleted
-  and/or eventually a "delete all" button

Also, since the marker list can be quite long, would it be possible to enlarge the marker table, maybe making the preferences window a two-sided window ?

Cheers,

Simon

**Comment by g4ixt** [comment]
**- allow multiple row selection to be deleted**
I don't know how to identify multiple selected rows so perhaps this would be hard.

**- and/or eventually a "delete all" button**
Yes, this is  easy.  I noticed this would be useful during testing but didn't add it because it was quite tricky getting the other bits working.  I will add it in future.

It takes quite a while to make a consistent, aligned, nice-looking window that re-sizes properly, but when I add more 'preferences' I will probably split it into different windows and make the table larger.

**Comment by g4ixt** [comment]
Included in v0.10.2
'Delete all' button added. 
Table made wider.
